website/src/controllers/payments/iap.js

```diff
@@ -24,15 +24,11 @@ exports.androidVerify = function(req, res, next) {
         ok: false,
         data: 'IAP Error'
       };
-    
-      console.error('IAP Setup ERROR');
-      console.error(error);
-        
-      res.json(resObj);
-        
-      return;
+
+      return res.json(resObj);
```

Comment:
Removed all the console things and collapsed the return and res.json calls. If logging is necessary, we should use the actual express logger.